keyboards/converter/usb_usb/custom_matrix.cpp

```diff
@@ -169,6 +178,7 @@ extern "C"
                 keyboard_set_leds(host_keyboard_leds());
             }
         }
+        matrix_scan_kb();
```

Comment:
This should actually be: 

```suggestion
        matrix_scan_quantum();
```

Reply:
Thanks, updated.